cl/people_db/lookup_utils.py

```diff
@@ -538,16 +538,16 @@ def lookup_judge_by_full_name(
     for filter_set in filter_sets:
         applied_filters.extend(filter_set)
         candidates = Person.objects.filter(*applied_filters)
-        if len(candidates) == 0:
+        if await candidates.acount() == 0:
             # No luck finding somebody. Abort.
             return None
-        elif len(candidates) == 1:
+        elif await candidates.acount() == 1:
             # Got somebody unique!
-            return candidates.first()
+            return await candidates.afirst()
```

Comment:
This will run the `acount()` twice in some instances. Let's put it above the first if statement, and then use the count in the `if` and `else` statements.

Reply:
applied as suggested